### DISCUSSION TITLE
High settings model count for simple page. Normal behavior?

### DISCUSSION THREAD

**Discussion by gavinhewitt** [spatie/laravel-settings#273] [Q&A]
Hi,

I have a fairly simple page and not too many settings, but when I check debugbar the model count for the settings is quite high and I have no idea why this is. Is this an indication something is not configured right? Or is it just how it works?

![image](https://github.com/spatie/laravel-settings/assets/1969103/60090317-870b-494e-841e-08064e981363)

MySQL crashes every few hours because of `too many connections` and I'm trying to determine if `laravel-settings` has anything to do with it. Probably not but I have to exclude every possibility :(

**Comment by rubenvanassche** [comment]
Can you provide me a demo repository illustrating the problem? Just tried it myself and everything seems to be working fine.
